src/actions/FioActions.tsx

```diff
@@ -62,7 +62,7 @@ export function checkFioObtData(wallet: EdgeCurrencyWallet, transactions: EdgeTr
     if (account == null || account.currencyConfig == null || fioWallets.length === 0) {
       checkFioObtDataCounter++
       return setTimeout(() => {
-        dispatch(checkFioObtData(wallet, transactions))
+        dispatch(checkFioObtData(wallet, transactions)).catch(err => console.error(err))
       }, 400)
     }
```

Comment:
This should be turned into a `while` loop, using `await snooze(400)`, to delay the rest of the function until the FIO wallets appear.

Reply:
No while loop is needed since this function is recursive.